extensions/src/main/java/org/pentaho/platform/web/http/api/resources/RepositoryResource.java

```diff
@@ -981,4 +1002,18 @@ public RepositoryDownloadWhitelist getWhitelist() {
   public void setWhitelist( RepositoryDownloadWhitelist whitelist ) {
     this.whitelist = whitelist;
   }
+
+  protected Response buildSafeHtmlServerErrorResponse( String msg ) {
+    return Response.serverError()
```

Comment:
Please see comment https://github.com/pentaho/pentaho-platform/pull/5131#discussion_r963118216, in which I try to demonstrate that the correct error for this situation is 403-Forbidden.

Reply:
I will update the code with your suggestion